modular_citadel/code/modules/projectiles/boxes_magazines/external/pistol.dm

```diff
@@ -78,3 +78,13 @@
 	build_path = /obj/item/ammo_box/a357
 	category = list("Ammo")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/sl357ap
+	name = "revolver speedloader (.357 AP)"
+	desc = "A revolver speedloader. Cuts through like a hot knife through butter."
+	id = "sl357ap"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 30000, MAT_TITANIUM = 45000)
```

Comment:
Man, sure some of these designs have pretty high material costs.

Reply:
nobody tell him about the manipulators

Reply:
Oh yea forgot about those, top-shelf ones can bring materials cost down to 25% of the original iirc.